keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroupTest.kt

```diff
@@ -413,7 +413,10 @@ object ActiveServerGroupTest : ModelParsingTestSupport<CloudDriverService, Activ
     image = ActiveServerGroupImage(
       imageId = ami,
       appVersion = "$app-3.16.0-h205.121d4ac",
```

Comment:
Here's an example of what the data looks like for debs. To find the baseOS, I look for either "xenial" or "bionic" as strings in the `name`, `description`, and `imageLocation` fields. It's excellent garbage hackery. For netflix, these files are autogenerated by the bakery so it's reliable.

Reply:
Yes we definitely will. I agree, not great. 

Do you have any ideas if the operating system is present in any field in any clouddriver response? I checked the server group and image response. I don't see the data surfaced anywhere in either of those or the spinnaker UI, or the internal edda link for the image. So, I'm thinking we just don't have the data. 

If we don't have the data, do you have any ideas on how to do this better?